src/main/java/ru/itmo/sd/deadliner2bot/model/Chat.java

```diff
@@ -0,0 +1,21 @@
+package ru.itmo.sd.deadliner2bot.model;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.util.Set;
+
+@Entity
+@Table(name = "chats")
+public class Chat {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "chat_id")
+    private BigInteger chatId;
+
+    @OneToMany(mappedBy = "chat")
+    private Set<Todo> todos;
```

Comment:
We need to iterate over todos in some specific order, so let's use `List` instead. `Set` would be convenient to check if some mapped entity exists.

Reply:
Speaking about daily notification, we will sort todos by end_time and maybe priority. It seems to me, for such purposes `Set` (`TreeSet` as implementation) is good.

Reply:
Then you need to explicitly tell JPA to user TreeSet as implementation.